Python/Product/Analysis/Interpreter/Default/CPythonInterpreter.cs

```diff
@@ -247,6 +307,9 @@ class GetModuleCallable {
             _searchPaths = _state.GetSearchPaths();
             _searchPathDb = null;
             _zipPackageCache = null;
+
+            BeginUpdateSearchPathPackages();
+
             ModuleNamesChanged?.Invoke(this, EventArgs.Empty);
```

Comment:
Do you still want to fire this event here, it will be done at the end of background task too.